kubevirt-hostpath-provisioner/kubevirt-hostpath-provisioner.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubevirt-hostpath-provisioner
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: kubevirt-hostpath-provisioner
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+provisioner: kubevirt.io/hostpath-provisioner
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer
```

Comment:
This is old way as per https://github.com/kubevirt/hostpath-provisioner#deployment doc looks like it should be https://github.com/kubevirt/hostpath-provisioner-operator/blob/main/deploy/storageclass-wffc-csi.yaml like this one